config/hubs/farallon.cluster.yaml

```diff
@@ -0,0 +1,182 @@
+name: farallon
+provider: kubeconfig
+kubeconfig:
+  file: secrets/farallon.yaml
+hubs:
+  - name: farallon-staging
+    domain: staging.farallon.2i2c.cloud
+    template: daskhub
+    auth0:
+      connection: github
+    config:
+      scratchBucket:
+        enabled: false
+      base-hub:
+        nfsPVC:
+          nfs:
+            # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+            mountOptions:
+            - rsize=1048576
+            - wsize=1048576 
+            - timeo=600
+            - soft # We pick soft over hard, so NFS lockups don't lead to hung processes
```

Comment:
when using `soft` they recommend `timeo` of 150

Reply:
They recommend *at least* 150, so I figured I'll keep it at the default

Reply:
Yep, I read too quickly...

Reply:
Now that solved the timeout, do you still want to keep the `soft` option?

Reply:
Yeah, definitely wanna keep it on soft.